src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -540,6 +542,8 @@ public AnnotationViewModel(WorkspaceViewModel workspaceViewModel, AnnotationMode
                 SetGroupOutPorts();
                 CollapseGroupContents(true);
             }
+
+            AnnotationModel.UpdateErrorAndWarningIconVisibility();
```

Comment:
Could this be on the AnnotationModel ctor instead?

Reply:
Moved